fulltext/backends/__ps.py

```diff
@@ -18,8 +18,10 @@
 
 
 def _get_file(f, **kwargs):
-    return run('pstotext', '-', stdin=f).decode('utf8')
+    out = run('pstotext', '-', stdin=f)
+    return out.decode(kwargs['encoding'], kwargs['encoding_errors'])
```

Comment:
Would it make sense to push the `encoding` and `encoding_errors` args to `run()` and do the decoding there?

Reply:
Mmm... it'd basically be the same but `run` would be a bit more complex because it'd have to accept encoding and errors as mandatory args. It's better to just do this in `handle_` functions where we already have them IMO.

Reply:
OK, I don't feel strongly about it. Just thought it might be a chance to reduce repetition.